client/src/modules/list/components/Items/model/reducer.js

```diff
@@ -31,7 +31,21 @@ const comments = (state = {}, action) => {
 
 const items = (state = {}, action) => {
   switch (action.type) {
-    case ItemActionTypes.ADD_SUCCESS:
+    case ItemActionTypes.ADD_SUCCESS: {
+      const {
+        payload: {
+          item,
+          item: { _id },
+          listState
+        }
+      } = action;
+      const items = { [_id]: item, ...state };
+
+      return {
+        ...items,
+        [_id]: { ...item, animate: shouldAnimate(items, item, listState) }
+      };
```

Comment:
you can just do this:
```

return {
        ...state,
        [_id]: { ...item, animate: shouldAnimate(items, item, listState) }
};
```

you'll spread existing items, and the add new one with animate property